### DISCUSSION TITLE
override keyword on my-element.ts

### DISCUSSION THREAD

**Discussion by bsastregx** [lit/lit#3444] [Q&A] [answered]
Hi there, I am new to Lit.

I have cloned the Typescript starter project.
There is a predefined element called **my-element** in it.
In the typescript file I see that for the styles and for the render method, the **override** keyword is present. This is not the case with the documentation on the lit webpage.

**my-element.ts (typescript project)**
![image](https://user-images.githubusercontent.com/54677414/201153969-f6537cee-3fc6-41a3-ac3f-8a9f205310b1.png)

**lit website example**
![image](https://user-images.githubusercontent.com/54677414/201154373-7fc79828-4db0-4d0c-b4d0-12bb1c2554c7.png)

What is the purpose of the override keyword? Why it is present on my-element.ts (typescript starter project), and not in the lit website example?

Thanks in advance.

**Comment by justinfagnani** [comment, marked as answer]
This is because of the `"noImplicitOverride": true` that we have in the tsconfig: https://www.typescriptlang.org/tsconfig#noImplicitOverride

I personally think it's not that useful and would vote to remove it from the starter kits. @rictic @kevinpschaaf @e111077

**Reply by kevinpschaaf** [reply]
Fine with me, seems less frictional for new users. People can always tighten up rules to their liking.

**Comment by bsastregx** [comment]
Thanks very much to both of you.
